queries.sql

```diff
@@ -71,3 +71,143 @@ select species,min(weight_kg),max(weight_kg) from animals group by species;
 -- Answer: Pokemon(3), Digimon(Null)
 -- Query:
 select species,avg(escape_attempts) from animals where date_part('year',date_of_birth) between 1990 and 2000 group by species;
+
+
+-- Day 3
+
+
+-- Animals owned by Melody Pond
+
+select
+owners.fullname, animals.name
+from animals
+join owners on animals.owner_id = owners.id
+where owners.fullname = 'Melody Pond';
+
+--   fullname   |    name    
+-- -------------+------------
+--  Melody Pond | Squirtle
+--  Melody Pond | Charmander
+--  Melody Pond | Blossom
+-- (3 rows)
+
+
+-- Animals who are Pokemon 
+
+select
+species.name, animals.name
+from animals
+join species on animals.species_id = species.id
+where species.name = 'Pokemon';
+
+--   name   |    name    
+-- ---------+------------
+--  Pokemon | Squirtle
+--  Pokemon | Charmander
+--  Pokemon | Blossom
+--  Pokemon | Pikachu
+-- (4 rows)
+
+
+-- Owner's animal count 
+
+select
+owners.fullname, count(animals.name)
+from owners
+left join animals on animals.owner_id = owners.id group by owners.fullname;
+
+--     fullname     | count 
+-- -----------------+-------
+--  Dean Winchester |     2
+--  Bob             |     2
+--  Sam Smith       |     1
+--  Jodie Whittaker |     0
+--  Jennifer Orwell |     2
+--  Melody Pond     |     3
+-- (6 rows)
```

Comment:
- [ ] Great work done here. The query should return the owners and names of the animals they own, not the count of animals that they have. Kindly change it and write the correct query. 👍 💻